src/js/utils/styles.js

```diff
@@ -208,18 +234,56 @@ export const focusStyle = css`
   }
 `;
 
+export const getInputPadBySide = (props, side) => {
+  if (typeof props.theme.global.input.padding !== 'object') {
+    return props.theme.global.input.padding;
```

Comment:
would that work if the `props.theme.global.input.padding` is a string of t-shirt sizes?

Reply:
Good call, I've updated to allow this to work.